docs/releases/4.2.txt

```diff
@@ -247,6 +247,35 @@ Miscellaneous
 Features deprecated in 4.2
 ==========================
 
+``index_together`` option deprecated in favor of ``indexes``
+------------------------------------------------------------
+
+The :attr:`~django.db.models.Options.index_together` option is deprecated in
+favor of the :attr:`Meta.indexes <django.db.models.Options.indexes>` option.
+
+Migrating existing ``index_together`` should be handled a migration.
+For example::
+
+    class Author(models.Model):
+        x = a
+        y = b
```

Comment:
Can we use a real field definitions rather than `a`, `b`? e.g. `CharField()`, `IntegerField()`.